Lib/idlelib/editor.py

```diff
@@ -433,6 +435,10 @@ def createmenubar(self):
         self.menudict['file'].insert_cascade(3, label='Recent Files',
                                              underline=0,
                                              menu=self.recent_files_menu)
+        if 'run' in self.menudict:
+            self.checkers_menu = Menu(self.menubar, tearoff=0)
+            self.menudict['run'].insert_cascade(3, label='Code Checkers',
+                                                menu=self.checkers_menu)
```

Comment:
I think this should be included in `mainmenu.py` and then the menu populated like helpmenu is in the lines just below this.  One thing though -- if there aren't any checkers, the menu item still appears, but hovering shows nothing.  I think there should be a default of `<None>` that's greyed out or something similar so that it looks like the hovering is working.